pkg/daemon/ceph/client/image.go

```diff
@@ -41,8 +41,11 @@ type CephBlockImage struct {
 	InfoName string `json:"name"`
 }
 
-func ListImages(context *clusterd.Context, clusterInfo *ClusterInfo, poolName string) ([]CephBlockImage, error) {
+func ListImages(context *clusterd.Context, clusterInfo *ClusterInfo, poolName, namespace string) ([]CephBlockImage, error) {
```

Comment:
For code readability, how about creating two methods? The latter will have exactly your implementation, and the first one just calls the second one.
```suggestion
func ListImagesInPool(context *clusterd.Context, clusterInfo *ClusterInfo, poolName string) ([]CephBlockImage, error) {
   return ListImagesInRadosNamespace(context, clusterInfo, poolName, "")
}
func ListImagesInRadosNamespace(context *clusterd.Context, clusterInfo *ClusterInfo, poolName, namespace string) ([]CephBlockImage, error) {
```